tests/cli/test_checkpoint.py

```diff
@@ -727,6 +731,82 @@ def test_checkpoint_new_raises_error_on_existing_checkpoint(
     )
 
 
+def test_checkpoint_new_no_fluent_datasource_messages(
+    caplog, monkeypatch, data_context_with_block_datasource
+):
+    """
+    What does this test and why?
+    The `checkpoint new` CLI flow should now print warnings/errors if all of the Datasources in the DataContext are configured with the block config style API.
```

Comment:
Sorry for the nit, just confused me the first time I read it.
```suggestion
    The `checkpoint new` CLI flow should not print warnings/errors if all of the Datasources in the DataContext are configured with the block config style API.
```